### ISSUE TITLE
Getting this error (DNN9.10.1 NB4.1.6)

### ISSUE THREAD

**Issue by trouble2** [openstore-ecommerce/OS_DnnIdx#6]
NBS Scheduler
OKOKOKOK Service Start. Failed. providername: OpenStore_DnnIdx Nevoweb.DNN.NBrightBuy.Providers Portal:Shop Error:System.TypeLoadException: Cannot load type Nevoweb.DNN.NBrightBuy.Providers from assembly OpenStore_DnnIdx, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null. bij System.Reflection.RuntimeAssembly.GetType(RuntimeAssembly assembly, String name, Boolean throwOnError, Boolean ignoreCase, ObjectHandleOnStack type) bij System.Reflection.RuntimeAssembly.GetType(String name, Boolean throwOnError, Boolean ignoreCase) bij System.Activator.CreateInstance(String assemblyString, String typeName, Boolean ignoreCase, BindingFlags bindingAttr, Binder binder, Object[] args, CultureInfo culture, Object[] activationAttributes, Evidence securityInfo, StackCrawlMark& stackMark) bij System.Activator.CreateInstance(String assemblyName, String typeName) bij Nevoweb.DNN.NBrightBuy.Components.NBrightBuyScheduler.DoWork(

**Comment by leedavi** [comment]
Is the assembly in the bin folder?

**Comment by skamphuis** [comment]
My guess is that you have something missing in your plugin configuration for the DnnIdx plugin:
![image](https://user-images.githubusercontent.com/4275042/159962976-48d70ed7-2474-4e18-af51-90bf6d21ff42.png)

**Comment by sboshuis** [comment]
Yes, we had the same issue and changing it to Nevoweb.DNN.NBrightBuy.Providers.DnnIdxProvider in plugin settings works just fine.
